interbotix_xs_toolbox/interbotix_xs_modules/interbotix_xs_modules/xs_robot/arm.py

```diff
@@ -187,11 +204,14 @@ def __init__(
             complete one move
         :param accel_time: (optional) time [s] it should take for all joints in the arm to
             accelerate/decelerate to/from max speed
+        :param update_kinematics: (optional) decides whether or not to update forward
+            kinematics of the arm. Can be disabled to save computations
```

Comment:
Add a note to this param stating that users can call `capture_joint_positions` or `_update_Tsb` directly if they want to update the FK.